agent/connectionchecker/connection_checker_test.go

```diff
@@ -121,7 +121,7 @@ func TestConnectionChecker(t *testing.T) {
 				Type:    inventorypb.ServiceType_MONGODB_SERVICE,
 				Timeout: durationpb.New(time.Nanosecond),
 			},
-			expectedErr: `server selection error: context deadline exceeded, current topology: \{ Type: Unknown, Servers: \[\{ Addr: 127.0.0.1:27017, Type: Unknown \}, \] \}`,
+			expectedErr: `context deadline exceeded`,
```

Comment:
it can be like follow I think
```suggestion
			expectedErr: `.*context deadline exceeded.*`,
```

Reply:
Yep, also should work